content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -158,29 +167,29 @@ const orders = await prisma.order.findMany({
 })
 ```
 
-You can also use a shorthand notation for this query, where you leave out the `is` and specify just the fields that you want to filter for:
+Use `equals` to filter for orders which match on all fields in the shipping address:
 
 ```ts
 const orders = await prisma.order.findMany({
   where: {
     shippingAddress: {
-      street: '555 Candy Cane Lane',
+      equals: {
+        street: '555 Candy Cane Lane',
+        city: 'Wonderland',
+        zip: '52337',
+      },
     },
   },
 })
 ```
 
-Use `equals` to filter for orders which match on all fields in the shipping address:
+You can also use a shorthand notation for this query, where you leave out the `equals` and specify just the fields that you want to filter for:
 
 ```ts
 const orders = await prisma.order.findMany({
   where: {
     shippingAddress: {
-      equals: {
-        street: '555 Candy Cane Lane',
-        city: 'Wonderland',
-        zip: '52337',
-      },
+      street: '555 Candy Cane Lane',
```

Comment:
@keerlu the shorthand notation is invalid here, all the required fields need to be passed to make it valid.

Which makes this `specify just the fields that you want to filter for:` invalid as well since you can't pass a subset of fields

Reply:
Oops, thanks, have removed the wrong description and accepted your change